subscription-engine/kafka-ws/package.json

```diff
@@ -39,7 +39,7 @@
     "supertest": "^4.0.2"
   },
   "dependencies": {
-    "@dojot/microservice-sdk": "^0.1.7",
+    "@dojot/microservice-sdk": "^0.1.8",
```

Comment:
Isn't it better to fix the version, instead of passing "^0.1.8"? Our "minor" versions can break things in this case.